lib/content-services/src/lib/search/components/search-date-range-advanced-tabbed/search-date-range-advanced-tabbed.component.html

```diff
@@ -0,0 +1,32 @@
+<adf-search-filter-tabbed
+    [settings]="settings"
+    [queries]="queries"
+    [valuesToDisplay]="valuesToDisplay"
+    (fieldsChanged)="onFieldsChanged($event)"
+    (displayedLabelsByFieldTranslated)="onDisplayLabelsByFieldsTranslated($event)"
+    (queriesCombined)="onQueriesCombined($event)"
+    (valuesToDisplayCombined)="onValuesToDisplayCombined($event)">
+    <ng-container *ngFor="let field of fields">
+        <adf-search-date-range-advanced
+            *adf-search-filter-tab="displayedLabelsByField[field]"
+            [dateFormat]="settings.dateFormat"
+            [maxDate]="settings.maxDate"
+            [field]="field"
+            [initialValue]="startValue"
+            (updatedDisplayValue)="onDisplayValueUpdate($event, field)"
+            (updatedQuery)="onQueryUpdate($event, field)"
+            (changed)="onDateRangedValueChanged($event, field)"
+            (valid)="onTabValid($event, field)">
+        </adf-search-date-range-advanced>
+    </ng-container>
+</adf-search-filter-tabbed>
+
+<div class="adf-facet-buttons" *ngIf="!settings?.hideDefaultAction">
```

Comment:
I think you don't need these buttons. It will render duplicated buttons. Buttons are added automatically  
<img width="772" alt="image" src="https://github.com/Alfresco/alfresco-ng2-components/assets/115619721/69a1ffd9-a15b-4f8d-9b8b-ee39dbf84dce">

Reply:
@swapnil-verma-gl FYI - I can't now somehow reproduce duplicated buttons, but still please remove them from HTML file because they seem redundant. Even after removing them buttons are still present because they are auto-added. So please remove buttons from that html

Reply:
@AleksanderSklorz The buttons were only added to allow usability of the component in case someone decides to use it in some other layout, and **not** the chip layout that is currently present in ACA/ADW. For e.g, the following UI is used in demo-shell when the user initially performs the search and doesn't click on the 'Switch to new Layout' button. 
![image](https://github.com/Alfresco/alfresco-ng2-components/assets/92505353/b36c2d26-6c8d-46fd-a0f5-93c87092b832)
The visibility of these buttons is driven by the hideDefaultActions boolean flag, coming from the SearchWidgetSettings interface. However, we can remove them if it is confirmed that the users would not use any other layout with these widgets

Reply:
I think that we shouldn't add code just in case someone might want to use it differently. If we won't use it in foreseeable future then it's a dead code that we need to maintain which doesn't really make sense in this case so I agree with @AleksanderSklorz that we should remove it